auditlog/models.py

```diff
@@ -305,6 +305,19 @@ def changes_display_dict(self):
             changes_display_dict[verbose_name] = values_display
         return changes_display_dict
 
+    @property
+    def additional_data(self):
+        data = None
+        try:
+            data = json.loads(self.additional_data_json)
+        except:
+            pass
+        return data
```

Comment:
This shouldn't be a blank exeption handling but instead checking for `json.JSONDecodeError` and return a default value of `{}`.
```suggestion
        try:
            return json.loads(self.additional_data_json)
        except json.JSONDecodeError:
            return {}
```